grails-app/views/profile/_lists.gsp

```diff
@@ -21,16 +21,37 @@
         <a name="{{listCtrl.readonly ? 'view_' : 'edit_'}}conservationStatus"></a>
 
         <div class="panel-body">
-            <div class="col-sm-2"><strong>Conservation status</strong></div>
-
-            <div class="col-sm-10">
-                <div class="col-sm-4" ng-repeat="status in listCtrl.conservationStatuses">
-                    <a href="${grailsApplication.config.collectory.base.url}/public/showDataResource/{{listCtrl.statusRegions[status.region].id}}"
-                       title="Threatened Species Codes - details" target="_blank">
-                        <div class="status"
-                             ng-class="listCtrl.getColourForStatus(status.status)">{{listCtrl.statusRegions[status.region].abbrev | default:'IUCN'}}</div>
-                        {{status.rawStatus}}
-                    </a>
+            <div class="row">
```

Comment:
Keep the bio statuses and conservation statuses in the same panel. Make the key of the status the 'heading' on the left, and the value the text on the right.